app/src/main/java/com/jmstudios/redmoon/Profile.kt

```diff
@@ -28,14 +29,16 @@ data class Profile(
         val color: Int,
         val intensity: Int,
         val dimLevel: Int,
-        val lowerBrightness: Boolean)
+        val lowerBrightness: Boolean,
+        val monochrome: Boolean = false)
     : Event, Comparable<Profile> {
```

Comment:
Does Kotlin support trailing commas now?
```suggestion
        val monochrome: Boolean = false,
    ) : Event, Comparable<Profile> {
```